_docs-v5/view-api/view-render-hooks.md

```diff
@@ -18,3 +18,14 @@ When the above hooks are specified as a function in the form `function(arg)`, th
 
 - `view` - a [View Object](view-object)
 - `el` - the view's root HTML element. not available in `viewClassNames`
+
+
+## View Changes
+
+There are 2 general rules that drives the call fo the above mentioned hooks:
+- `viewDidMount` and `viewWillUnmount` hooks are only called when changing [views of different types](view-object#view-types)
+- `viewClassNames` hook however is called no mather what the view change occurs
+
+A couple of example of view type changes and their outcomes:
```

Comment:
The phrase `A couple of example of view type changes` is grammatically incorrect